docs/user-guide/reading_writing_data.rst

```diff
@@ -105,6 +105,15 @@ You can now create the :py:class:`~hermes_core.timedata.TimeData` object,
     >>> from hermes_core.timedata import TimeData
     >>> timedata = TimeData(data=ts, meta=input_attrs)
 
+The :py:class:`~hermes_core.timedata.TimeData` object also accepts optional arguments for loading
+support data and non-record-varying (NRV) data. These arguments are required to each be a `dict`
+of either :py:class:`~astropy.units.Quantity` or :py:class:`~astropy.table.Column` objects.
```

Comment:
Not sure that a column object is really necessary as it contains table functionality that is not needed. How about we accept python arrays and numpy arrays?

Reply:
I liked Columns because they have a `.meta` attribute built in through astropy.

Reply:
Updated to NDData